### DISCUSSION TITLE
Simple OpenAI bot allowing user to upload file

### DISCUSSION THREAD

**Discussion by pascalvallet** [FlowiseAI/Flowise#1629] [Q&A] [answered]
I am trying to find a way to allow the user to uplaod file when using the openAI chatbot but I cannot see how. My flow is vry simple, I am a beginner!
![CleanShot 2024-01-28 at 14 44 27@2x](https://github.com/FlowiseAI/Flowise/assets/15332281/f8279c34-9d60-4a01-a6e6-5e178fdb6b15)
Thanks for your help

**Comment by rvy474** [comment, marked as answer]
This does not exist in Flowise from what Ive found.

**Reply by pascalvallet** [reply]
Too bad! This would be very useful.

**Reply by HenryHengZJ** [reply]
will be adding support soon!

**Reply by pascalvallet** [reply]
Wonderful I cannot wait.

**Reply by octavioccl** [reply]
yeah, that would be great to have it, any idea when this could be released @HenryHengZJ? I was thinking of implementing some POCs based on that functionality but maybe I can wait for it

**Reply by octavioccl** [reply]
one feature that would be great is if we could pass the uploaded doc for example to the OpenAI Assistant, is that something the team has in mind?

**Reply by pascalvallet** [reply]
Any idea when it will be available ?

**Comment by roscoe1912** [comment]
would be great, waiting for it

**Comment by ori-aventure** [comment]
Waiting, too!

**Comment by pascalvallet** [comment]
Any update on this feature ?
